.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "wcwave_adaptors"]
-	path = wcwave_adaptors
-	url = https://github.com/tri-state-epscor/wcwave_adaptors.git
```

Comment:
Sorry if I made a mistake by adding wcave_adaptors. Actually, no. I haven't made any changes. The only difference is that wcwave_adaptors is inside wcwave_adaptors submodule. Also, I added default.conf in wcwave_adaptors.